src/components/inputs/FieldMessage.tsx

```diff
@@ -22,7 +22,7 @@ export default function FieldMessage({
     <div id={id} aria-live="polite" aria-atomic="true">
       <span
         className={cn(
-          "flex items-center h-4 gap-1 text-[13px] font-medium text-base-300",
+          "flex items-center h-4 gap-1 py-4 text-[13px] font-medium text-base-300",
```

Comment:
@Dan-Y-Ko  Maybe better we make an error text shorter: "Whole numbers only"? That's a lot of padding. If not I think it's better to simply change `Average Hour/Sprint` column/row width in `TeamMember.tsx`. I can do it if you want.

Reply:
If we remove this padding, `Button` styles need to be updated in `EditHours.tsx`, `mb-10` needs to be changed to `mb-6`